lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -177,16 +200,25 @@ class Command(BaseCommand):
             default=False,
             help='The amount of credits awarded for course completion.',
         ),
+        make_option(
+            '-r',
+            '--new_report',
+            metavar='NEW_REPORT',
```

Comment:
the variable name NEW_REPORT does not correctly describe what we are doing here. Why not INCLUDE_GRADES? or something similar